cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -143,6 +148,23 @@ function ensure_addons() {
   log INFO "== Kubernetes addon ensure completed at $(date -Is) =="
 }
 
+function is_leader() {
```

Comment:
This real pokes around at the internals. We want to start moving the addon manager to use the front door. Can you look at the controller-manager leader election result instead?

Reply:
The leader election results for kube-controller-manager are stored on it's endpoints object:

kubectl -n kube-system get ep kube-controller-manager -o json

Reply:
Thanks for pointing this one! Done.